packages/integration_test/lib/common.dart

```diff
@@ -297,6 +297,9 @@ abstract class CallbackManager {
   /// Android only. Converts the Flutter surface to an image view.
   Future<void> convertFlutterSurfaceToImage();
 
+  /// Android only. Converts the image view to the Flutter surface.
+  Future<void> revertFlutterImage();
```

Comment:
Why are we adding implementation details to a public API?

What is the use case for toggling between an Image backed surface and a non-Image backed surface in a test? Flutter developers cannot do this from their apps.

Reply:
First of all i want to point out that this function was mentioned 3 years ago in the documentation of another function, but was not made publicly available. That's what this PR is for, making the function accessible.

The reason why we need this, is the "takeScreenshot()" function in Integration tests. It is required on Android (only) to "convertFlutterSurfaceToImage()" being called first, before it can take the screenshot. 
Unfortunately, you have to call "revertFlutterImage()" (which is implemented in this PR) to go further with the Integration Test.

That's basically all. I guess the main use case is to write an integration test that runs through all screens of the app and takes screenshots in different formats so that they can be used for Stores. Without being able to call revertFlutterImage() manually, the integration test cannot be performed on Android devices because they fail without converting the image back.